src/output/help.rs

```diff
@@ -12,7 +12,6 @@ use crate::{
     build::{arg::display_arg_val, App, AppSettings, Arg, ArgSettings},
     output::{fmt::Colorizer, Usage},
     parse::Parser,
-    util::VecMap,
```

Comment:
At this point `VecMap` should be unused and we should be able to remove it as a dependency

Reply:
It is used in `default_vals_ifs` for `Arg`

Reply:
Huh, somehow overlooked that in my search but that looks to be overkill.  All it does is append and then we iterate with indexes.  That can be replaced with a `Vec`.

Creating a PR to address this.

Reply:
Please do give it a careful look. We had some issues with Vec IIRC.